tests/Fixtures/is.gd/shorten.http-400.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
In this case maybe it would be better to use `golden` files instead of `fixtures`.

Reply:
Will look into this